src/main/proto/node.proto

```diff
@@ -3,31 +3,22 @@ package com.twcable.grabbit.proto;
 option java_package = "com.twcable.grabbit.proto";
 option java_outer_classname = "NodeProtos";
 
-/*
```

Comment:
None of this was necessary for conforming to the new version of protoc, but thought it would be nice to consolidate our messaging a bit, and use some of the new features. 

One thing that was sort of curious, was that I was expecting as a sort of unimportant side-effect to find a byte savings per node, and another byte savings per multi-value property with this restructure based on my understanding of the wire encoding. 

Ignoring the other stuff, if we were to send a single property with the old encoding, I would expect one byte to indicate the wire type and field value for properties (0x12), proceeded by another byte for the property header, which I thought would be removed with this PR (0x0A), then so on and so forth for property fields. 

When I did a before and after wget to /grabbit/content?path=/test/path, I received the same number of bytes before and after. Wondering if there is another optimization going on for single field embedded messages, or if I don't understand the full picture. 

Anyhoo..